src/Maestro/tests/Maestro.ScenarioTests/EndToEndFlowLogic.cs

```diff
@@ -38,6 +49,35 @@ public EndToEndFlowLogic(TestParameters parameters)
             childSourceBuildAssets = GetAssetData("Baz", "1.3.0", "Bop", "1.0");
             childSourceAssets = GetSingleAssetData("Baz", "1.3.0");
 
+            feedFlowSourceAssets = ImmutableList.Create(
```

Comment:
It feels like this should be in the calling test.  This class is just becoming "all the tests" class. Which makes it REALLY hard to comprehend in any really useful manner.